packages/image_picker/image_picker_macos/macos/image_picker_macos.podspec

```diff
@@ -0,0 +1,23 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint image_picker_macos.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'image_picker_macos'
+  s.version          = '0.0.1'
+  s.summary          = 'Flutter plugin that shows an image picker.'
+  s.description      = <<-DESC
+A Flutter plugin for picking images from the image library, and taking new pictures with the camera.
+Downloaded by pub (not CocoaPods).
+                       DESC
+  s.homepage         = 'https://github.com/flutter/packages'
+  s.license          = { :type => 'BSD', :file => '../LICENSE' }
+  s.author           = { 'Flutter Team' => 'flutter-dev@googlegroups.com' }
```

Comment:
Nit: Some packages use a different name (e.g. [podspec of `image_picker_ios`](https://github.com/flutter/packages/blob/6c4482a64492e1cc5980f856bec36fb4d96fa983/packages/image_picker/image_picker_ios/ios/image_picker_ios.podspec#L14)).